src/toast/todmap/mapmaker.py

```diff
@@ -144,7 +144,7 @@ def get_steps_and_preconditioner(self):
             tod = obs["tod"]
             common_flags = tod.local_common_flags(self.common_flags)
             common_flags = (common_flags & self.common_flag_mask) != 0
-            if self.intervals and self.intervals in obs:
+            if (self.intervals is not None) and (self.intervals in obs):
```

Comment:
I can't decide if redundant parentheses improve readability or add clutter.

Reply:
I find long logical expressions without parenthesis difficult to parse visually.  That may just be a personal issue I have.  Feel free to remove them.  The main thing I care about is using 
```
if A is not None:
```
rather than
```
if A:
```
Since those are actually different things.